estate/models/__init__.py

```diff
@@ -0,0 +1 @@
+from . import estate_property, estate_property_misc
```

Comment:
```suggestion
from . import estate_property
from . import estate_property_misc
```
We usually do one import per line in those init files 🙂